### DISCUSSION TITLE
CORS error after setup

### DISCUSSION THREAD

**Discussion by wilburx9** [plausible/analytics#1042] [Self-Hosted Support] [answered]
After setup, it seems not to be working because of a CORS error:

> Failed to load resource: net::ERR_FAILED (index):1 Access to
> XMLHttpRequest at 'https://example.com/api/event' from origin
> 'https://www.example.com' has been blocked by CORS policy: No
> 'Access-Control-Allow-Origin' header is present on the requested
> resource. example.com/api/event:1 Failed to load resource:
> net::ERR_FAILED

Unlike [this solved issue](https://github.com/plausible/analytics/discussions/1015), mine doesn't seem to be caused by authentication issues.

Plausible script in the header of example.com:

    <script async defer="defer" data-domain="example.com" src="https://stats.example.com/js/plausible.js"></script>

Nginx Plausible server block:

    server {
    server_name stats.example.com www.stats.example.com;
    
            location / {
                    proxy_pass http://127.0.0.1:8000;
                    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            }
    
        listen [::]:443 ssl ipv6only=on; # managed by Certbot
        listen 443 ssl; # managed by Certbot
        ssl_certificate /etc/letsencrypt/live/stats.example.com/fullchain.pem; # managed by Certbot
        ssl_certificate_key /etc/letsencrypt/live/stats.example.com/privkey.pem; # managed by Certbot
        include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
        ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
    
    
    }
    server {
    if ($host = www.stats.example.com) {
    return 301 https://$host$request_uri;
    } # managed by Certbot
    
    
        if ($host = stats.example.com) {
            return 301 https://$host$request_uri;
        } # managed by Certbot
    
    
            server_name stats.example.com www.stats.example.com;
    
            listen 80;
            listen [::]:80;
        return 404; # managed by Certbot
    
    
    
    
    }

I think I messed somewhere in the setup but I still can't tell after several hours of debugging.

**Comment by wilburx9** [comment, marked as answer]
So the issue was that I used **https://example.com** instead of **https://stats.example.com** in the configuration file 🤦🏾‍♂️.

**Reply by metmarkosaric** [reply]
thanks for the update!
